dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/servlet/FileServletWrapper.java

```diff
@@ -1121,16 +1118,6 @@ public boolean isAvailable() {
   // end 268176 Welcome file wrappers are not checked for resource existence
   // WAS.webcontainer
 
-  // Begin PK27620
-
-    public void nameSpacePostInvoke() {
-    this.webAppNameSpaceCollab.postInvoke();
-  }
-
-    public void nameSpacePreInvoke() {
-    this.webAppNameSpaceCollab.preInvoke(context.getModuleMetaData().getCollaboratorComponentMetaData());
-  }
-
```

Comment:
@jhanders34  what is the rationale for this removal? I don't see anyone calling these and the package isn't exported as SPI as far as I can tell so I don't see a problem here removing it. I just want to be clear that you're removing because no one currently invokes them? By removing we reduce having to load those classes etc. Or is there another reason?

Reply:
Yeah it is to remove loading of the collaborator when this class loads.